drivers/flickr/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Flickr",
+    "description": "A Driver to synchronize your photos and videos with a Flickr account. In order to use it, you need to retrieve Oauth tokens with the provided get_tokens.py script.",
+    "options": {
+	"oauth_token": {
+	    "name": "Oauth token",
+	    "description": "Your Oauth token retrieved with get_tokens.py",
+	    "type": "string"
+	},
+	"oauth_token_secret": {
+	    "name": "Oauth token secret",
+	    "description": "Your Oauth token secret retrieved with get_tokens.py",
+	    "type": "string"
+	},
+	"changes_timer": {
+	    "name": "Changes timer",
+	    "description": "The amount of time you want Onitu to wait between each check for new files on your Flickr account, in seconds. Defaults to 30.",
+	    "type": "integer",
+	    "default": 30
```

Comment:
I think we should choose a saner default. Something like '120' or even '300' should be enough.